submissions/DmitryHniezdilov/task-js-dom/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.min.css" />
+    <title>Task js dom</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;600&display=swap&_v=20220825093305"
+      rel="stylesheet"
+    />
+    <link rel="icon" type="image/x-icon" href="./img/logo-mountain.svg" />
+  </head>
+
+  <body>
```

Comment:
You can [combine gradient with an image](https://stackoverflow.com/questions/46252818/how-to-combine-css-linear-gradient-with-images) for one element.